### ISSUE TITLE
Sageplot artifacts

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/precalculus#165]
Some of the sageplots have some weirdness going on.  For example, [Activity 3.1.11](https://teambasedinquirylearning.github.io/precalculus/2024/LF1.html#LR-4-Investigate-slope)  jumps over the axis

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/30658947/e021b375-e468-4cdb-a025-602f302b09e5)

This is generated by the code
```
f(x) = 3*x-4
p=plot(f, (x, -10, 10), ymin=-10, ymax=10, color='blue', thickness=3, detect_poles=True, gridlines=[[-10..10],[-10..10]])
p
```

I copied the code into a sagecell, and it looks fine.  I built it locally, and it looks fine.  Is this possibly related to #123?

**Comment by StevenClontz** [comment]
Best guess: Sagecell produces a PNG, we're building SVG for the book. Did you build SVG or PNG locally?

**Comment by siwelwerd** [comment]
Here's what I see in the output:
```
successful application of /Users/drew/.ptx/xsl/extract-sageplot.xsl
converting AROC-linear-function.sage to AROC-linear-function.svg
```

So I think building a SVG?

**Comment by StevenClontz** [comment]
There goes my theory! So when you say "build locally" you mean build the whole PreTeXt document locally. Then maybe it's a Sage version issue. What version of Sage are you using?

**Comment by siwelwerd** [comment]
```
(base) drew@Drews-MBP precalculus % sage --version
SageMath version 9.7, Release Date: 2022-09-19
```

**Comment by StevenClontz** [comment]
My guess: there's a bug in `detect_poles` on the version of Sage (9.2) that's installed in our codespace (perhaps an upstream PreTeXt-CLI issue). But we don't have any asymptotes here so for now it's easiest to just remove that keyword argument.

**Comment by siwelwerd** [comment]
Actually there are a few more in some other sections (I see one in 2.3 for example).  Reopening until one of us can do a full audit of the usage of `detect_poles`.

**Comment by StevenClontz** [comment]
Ah I should have just done that. I'll knock it out now.

**Comment by siwelwerd** [comment]
Closed by #170
